Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,51 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_installed_python",
+            category=['misc'],
+            prompt="I installed Python",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_installed_python:
+    m 1sub "You did?"
+    m 1hub "That's great, [player]!"
+    m 1eub "What did you download it for?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "What did you download it for?{fast}"
+        "A personal project.":
+            m 1hub "How fun!"
+            m 3hub "Good luck with whatever you're programming!"
+            m 3ekbfa "Maybe you could help me out with a few things?"
+            m 1eub "I'll teach you as much as I can!"
```

Comment:
It doesn't sound right to me to have Monika suggest you helping her with a few things, and then offer to teach you as much as she can. Who is helping who in this case?

Reply:
Ack! My intent was to make it sound like Monika was going to teach the player and THEN the player could help her with MAS. I'll reword that.

Reply:
It should look a bit better, now.